sonar-helm-for-iac/converters/io-handling.go

```diff
@@ -65,6 +64,12 @@ func (s StdinReader) ReadInput(scanner *bufio.Scanner) (string, Files, error) {
 				if err != nil {
 					return mo.Err[[]byte](err)
 				}
+				if length == 0 {
+					fmt.Fprintf(os.Stderr, "Reading 0 lines of file %s from stdin\n", name)
+					// read new line and ignore it
```

Comment:
```suggestion
					// read new line and ignore it. HelmEvaluator writes a single empty line for empty files
```
I thought it was not obvious why there should be a line in this case :)